### PR TITLE
Fix print configuration

### PR DESCRIPTION
**Pull request by sbrunner**

### CODE REVIEW

**Review by ger-benjamin — APPROVED** [review]
Let's try

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/env.default_tmpl [hunk 1]
```diff
@@ -16,17 +16,19 @@ AUTHTKT_HTTP_ONLY=True
 AUTHTKT_SECURE=True
 AUTHTKT_SAMESITE=Lax
 BASICAUTH=False
-GEOPORTAL_INTERNAL_URL=http://geoportal:8080
 MUTUALIZED_PRINT_URL=mutualized-print.apps.openshift-ch-1.camptocamp.com
+# For internal print
+# DISABLE_LOCAL_PRINT will be set to # on mutialise print to comments thoes configuration
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/env.default_tmpl:21]:
```suggestion
# DISABLE_LOCAL_PRINT will be set to # on mutualised print to comment local configuration
```

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/env.default_tmpl [hunk 2]
```diff
@@ -16,17 +16,19 @@ AUTHTKT_HTTP_ONLY=True
...
+# DISABLE_MUTUALIZED_PRINT is set to # on local print to comments thoes configuration
```
Comment (arnaud-morvan) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/env.default_tmpl:22]:
```suggestion
# DISABLE_MUTUALIZED_PRINT is set to # on local print to comment mutualized configuration
```

**Review by arnaud-morvan — APPROVED** [review]
Minor comments

**Comment by sbrunner** [comment]
Thanks :-)

**Pull request merged**